aws/v18.2.1/release.yaml

```diff
@@ -74,7 +74,7 @@ spec:
   - name: kubernetes
     version: 1.23.15
   date: "2023-03-02T10:06:00Z"
-  state: active
+  state: deprecated
```

Comment:
technically we don't do this normally. this is another minor so shouldn't be deprecated

Reply:
Right, I reverted it 👍🏻